pkgs/by-name/wi/wifitui/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "wifitui";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "shazow";
+    repo = "wifitui";
+    tag = "v${version}";
+    hash = "sha256-JFs+7MDc0/hIDrefSRLWXurwJvvpR7LHJmCvmO1lpHA=";
+  };
+
+  vendorHash = "sha256-SEQPc13cefzT8SyuD3UmNtTDgcrXUGTX54SBrnOHJJw=";
+
+  ldflags = [
+    "-X main.Version=${version}"
+  ];
```

Comment:
It's also pretty common to use `-s` and `-w`, but it's up to you if you want those flags.